aequilibrae/project/database_specification/transit/tables/fare_zones.sql

```diff
@@ -4,12 +4,8 @@
 --@ **fare_zone_id** identifies the fare zone for a stop
 --@ 
 --@ **transit_zone** identifies the TAZ for a fare zone
---@ 
---@ **agency_id** identifies the agency fot the specified route
 
 CREATE TABLE IF NOT EXISTS fare_zones (
-	fare_zone_id    INTEGER  PRIMARY KEY,
-	transit_zone    TEXT     NOT NULL,
-	agency_id       INTEGER  NOT NULL,
```

Comment:
Why this? We CAN have different fare zones for different operators

Reply:
Let's make that fare_zone_id is names transit_fare_zone
And that should be something like 100,000 * agency_id + actual fare zone (or some counter for it)

Reply:
Let's allow for it to be empty in case there are multiple feeds inside that GTFS